### ISSUE TITLE
File upload/download unreliability

### ISSUE THREAD

**Issue by TacoByte** [NebulousLabs/Sia#1320]
referenced from https://github.com/NebulousLabs/Sia-UI/issues/336

I have been trying to upload some rare pepes as a test. These uploads have been going on for hours. Clearly they are very small files and shouldn't take that long. The transfers themselves aren't slow, it just seems that it's struggling to connect to hosts and get those pieces distributed. All kinds of errors are being logged.

Those do show up in `renter list`. However, attempting to download them returns popup `Download failed: couldn't fetch enough pieces to recover data.`

![uploads](https://cloud.githubusercontent.com/assets/8594622/16458868/0aca5bec-3e21-11e6-87e0-2e99d16ac273.png)

```
Renter info:
	Storage Spending:  717.5 mS
	Upload Spending:   21.02 SC
	Download Spending: 0 H
	Unspent Funds:     4.146 KS
	Total Allocated:   4.167 KS

Tracking 6 files:
 252.0 KB  1453756944023.png
   6.1 KB  untitled folder 3/.DS_Store
   7.8 KB  untitled folder 3/1453756843707.jpg
 252.0 KB  untitled folder 3/1453756944023.png
   8.0 KB  untitled folder 3/untitled folder/1443814339804.jpg
  56.4 KB  untitled folder 3/untitled folder/1452160563683.jpg
```

```
2016/06/29 13:22:50.433609 repair.go:278: repairing 1 chunks of untitled folder 3/1453756944023.png
2016/06/29 13:33:18.582092 repair.go:301: failed to upload to host 85.114.128.125:9982: contract has insufficient collateral to support upload
2016/06/29 13:33:18.582918 repair.go:301: failed to upload to host 82.220.99.82:9982: host is not accepting contracts
2016/06/29 13:33:18.583120 repair.go:301: failed to upload to host 108.61.96.209:9982: host is not accepting contracts
2016/06/29 13:33:18.583246 repair.go:301: failed to upload to host 97.107.52.227:9982: host is not accepting contracts
2016/06/29 13:33:18.583437 repair.go:301: failed to upload to host 216.84.97.66:9982: host is not accepting contracts
2016/06/29 13:33:18.583634 repair.go:301: failed to upload to host dataspace.ddns.net:9982: host is not accepting contracts
2016/06/29 13:33:18.583745 repair.go:301: failed to upload to host sia.tech.tedx.ee:9982: host is not accepting contracts
2016/06/29 13:33:18.583822 repair.go:301: failed to upload to host 198.211.103.40:9982: host is not accepting contracts
2016/06/29 13:33:18.583904 repair.go:301: failed to upload to host 112.74.97.188:9982: write tcp 192.168.1.4:57810->112.74.97.188:9982: write: broken pipe
2016/06/29 13:33:18.584007 repair.go:301: failed to upload to host 146.90.15.59:9982: write tcp 192.168.1.4:58278->146.90.15.59:9982: write: broken pipe
2016/06/29 13:33:18.584102 repair.go:301: failed to upload to host 73.69.100.22:9982: write tcp 192.168.1.4:58260->73.69.100.22:9982: write: broken pipe
2016/06/29 13:33:18.584204 repair.go:301: failed to upload to host cmoser.duckdns.org:9982: write tcp 192.168.1.4:58320->66.41.100.180:9982: write: broken pipe
2016/06/29 13:33:18.584262 repair.go:301: failed to upload to host 31.47.126.128:9982: write tcp 192.168.1.4:58334->31.47.126.128:9982: write: broken pipe
2016/06/29 13:33:18.584317 repair.go:301: failed to upload to host 64.137.228.210:9982: write tcp 192.168.1.4:58352->64.137.228.210:9982: write: broken pipe
2016/06/29 13:33:18.584371 repair.go:301: failed to upload to host alpha.sia.aspectron.com:9982: write tcp 192.168.1.4:58384->24.37.11.107:9982: write: broken pipe
2016/06/29 13:33:18.584427 repair.go:301: failed to upload to host sia.pylon.org.uk:10982: write tcp 192.168.1.4:58399->83.76.19.197:10982: write: broken pipe
2016/06/29 13:33:18.584481 repair.go:301: failed to upload to host 52.26.159.234:9982: write tcp 192.168.1.4:58400->52.26.159.234:9982: write: broken pipe
2016/06/29 13:33:18.584534 repair.go:301: failed to upload to host 138.201.13.159:9982: write tcp 192.168.1.4:58413->138.201.13.159:9982: write: broken pipe
2016/06/29 13:33:18.596950 repair.go:278: repairing 1 chunks of untitled folder 3/untitled folder/1443814339804.jpg
2016/06/29 13:35:51.885619 repair.go:301: failed to upload to host sia.internet-box.ch:9982: host did not accept revision: proposed file contract revision does not correctly add to the host's valid proof output
2016/06/29 13:35:51.898125 repair.go:278: repairing 1 chunks of untitled folder 3/untitled folder/1452160563683.jpg
2016/06/29 13:35:51.898322 repair.go:278: repairing 1 chunks of 1453756944023.png
2016/06/29 13:35:51.898447 repair.go:278: repairing 1 chunks of untitled folder 3/.DS_Store
2016/06/29 13:35:51.898557 repair.go:278: repairing 1 chunks of untitled folder 3/1453756843707.jpg
2016/06/29 13:35:56.899967 repair.go:278: repairing 1 chunks of 1453756944023.png
2016/06/29 13:46:04.503900 repair.go:301: failed to upload to host 198.211.103.40:9982: host is not accepting contracts
2016/06/29 13:46:04.511683 repair.go:301: failed to upload to host 97.107.52.227:9982: host is not accepting contracts
2016/06/29 13:46:04.511765 repair.go:301: failed to upload to host sia.tech.tedx.ee:9982: host is not accepting contracts
2016/06/29 13:46:04.511830 repair.go:301: failed to upload to host 216.84.97.66:9982: host is not accepting contracts
2016/06/29 13:46:05.353938 repair.go:301: failed to upload to host 108.61.96.209:9982: host is not accepting contracts
2016/06/29 13:46:06.695103 repair.go:301: failed to upload to host 82.220.99.82:9982: host is not accepting contracts
2016/06/29 13:46:06.695306 repair.go:301: failed to upload to host dataspace.ddns.net:9982: host is not accepting contracts
2016/06/29 13:46:06.695435 repair.go:301: failed to upload to host 52.26.159.234:9982: write tcp 192.168.1.4:54164->52.26.159.234:9982: write: broken pipe
2016/06/29 13:46:06.695535 repair.go:301: failed to upload to host 138.201.13.159:9982: write tcp 192.168.1.4:54177->138.201.13.159:9982: write: broken pipe
2016/06/29 13:46:06.695619 repair.go:301: failed to upload to host alpha.sia.aspectron.com:9982: write tcp 192.168.1.4:54178->24.37.11.107:9982: write: broken pipe
2016/06/29 13:46:06.695697 repair.go:301: failed to upload to host 146.90.15.59:9982: write tcp 192.168.1.4:54277->146.90.15.59:9982: write: broken pipe
2016/06/29 13:46:06.695775 repair.go:301: failed to upload to host cmoser.duckdns.org:9982: write tcp 192.168.1.4:54264->66.41.100.180:9982: write: broken pipe
2016/06/29 13:46:06.706669 repair.go:278: repairing 1 chunks of untitled folder 3/.DS_Store
2016/06/29 13:56:15.950334 repair.go:301: failed to upload to host 85.114.128.125:9982: contract has insufficient collateral to support upload
2016/06/29 13:56:15.951832 repair.go:301: failed to upload to host 112.74.97.188:9982: write tcp 192.168.1.4:63743->112.74.97.188:9982: write: broken pipe
2016/06/29 13:56:15.957809 repair.go:278: repairing 1 chunks of untitled folder 3/1453756843707.jpg
2016/06/29 14:02:34.430561 repair.go:301: failed to upload to host sia.internet-box.ch:9982: write tcp 192.168.1.4:64015->85.7.252.119:9982: write: broken pipe
2016/06/29 14:02:34.431319 repair.go:301: failed to upload to host 31.47.126.128:9982: host did not accept revision: proposed file contract revision does not correctly add to the host's valid proof output
2016/06/29 14:02:34.431385 repair.go:301: failed to upload to host 64.137.228.210:9982: host did not accept revision: proposed file contract revision does not correctly add to the host's valid proof output
2016/06/29 14:02:34.431481 repair.go:301: failed to upload to host 73.69.100.22:9982: host did not accept revision: proposed file contract revision does not correctly add to the host's valid proof output
2016/06/29 14:02:34.437053 repair.go:278: repairing 1 chunks of untitled folder 3/1453756944023.png
2016/06/29 14:03:41.403244 repair.go:278: repairing 1 chunks of untitled folder 3/untitled folder/1443814339804.jpg
2016/06/29 14:04:46.155615 repair.go:278: repairing 1 chunks of untitled folder 3/untitled folder/1452160563683.jpg
2016/06/29 14:05:46.727663 repair.go:278: repairing 1 chunks of untitled folder 3/untitled folder/1443814339804.jpg
2016/06/29 14:16:12.307002 repair.go:301: failed to upload to host 85.114.128.125:9982: contract has insufficient collateral to support upload
2016/06/29 14:16:12.315244 repair.go:301: failed to upload to host 198.211.103.40:9982: host is not accepting contracts
2016/06/29 14:16:12.315343 repair.go:301: failed to upload to host 216.84.97.66:9982: host is not accepting contracts
2016/06/29 14:16:12.315411 repair.go:301: failed to upload to host 82.220.99.82:9982: host is not accepting contracts
2016/06/29 14:16:12.315475 repair.go:301: failed to upload to host sia.tech.tedx.ee:9982: host is not accepting contracts
2016/06/29 14:16:12.315540 repair.go:301: failed to upload to host 97.107.52.227:9982: host is not accepting contracts
2016/06/29 14:16:12.315602 repair.go:301: failed to upload to host dataspace.ddns.net:9982: host is not accepting contracts
2016/06/29 14:16:12.315660 repair.go:301: failed to upload to host 108.61.96.209:9982: host is not accepting contracts
2016/06/29 14:16:12.315790 repair.go:301: failed to upload to host sia.internet-box.ch:9982: write tcp 192.168.1.4:49726->85.7.252.119:9982: write: broken pipe
2016/06/29 14:16:12.315870 repair.go:301: failed to upload to host 112.74.97.188:9982: write tcp 192.168.1.4:49713->112.74.97.188:9982: write: broken pipe
2016/06/29 14:16:12.315981 repair.go:301: failed to upload to host cmoser.duckdns.org:9982: write tcp 192.168.1.4:49782->66.41.100.180:9982: write: broken pipe
2016/06/29 14:16:12.316029 repair.go:301: failed to upload to host 31.47.126.128:9982: write tcp 192.168.1.4:49802->31.47.126.128:9982: write: broken pipe
2016/06/29 14:16:12.316074 repair.go:301: failed to upload to host 73.69.100.22:9982: write tcp 192.168.1.4:49803->73.69.100.22:9982: write: broken pipe
2016/06/29 14:16:12.316118 repair.go:301: failed to upload to host 52.26.159.234:9982: write tcp 192.168.1.4:49821->52.26.159.234:9982: write: broken pipe
2016/06/29 14:16:12.316164 repair.go:301: failed to upload to host 138.201.13.159:9982: write tcp 192.168.1.4:49834->138.201.13.159:9982: write: broken pipe
2016/06/29 14:16:12.316209 repair.go:301: failed to upload to host 64.137.228.210:9982: write tcp 192.168.1.4:49840->64.137.228.210:9982: write: broken pipe
2016/06/29 14:16:12.316254 repair.go:301: failed to upload to host alpha.sia.aspectron.com:9982: write tcp 192.168.1.4:49859->24.37.11.107:9982: write: broken pipe
2016/06/29 14:16:12.319669 repair.go:278: repairing 1 chunks of untitled folder 3/untitled folder/1452160563683.jpg
2016/06/29 14:17:05.546373 repair.go:278: repairing 1 chunks of 1453756944023.png
2016/06/29 14:17:58.033653 repair.go:301: failed to upload to host 146.90.15.59:9982: host did not accept revision: proposed file contract revision does not correctly add to the host's valid proof output
2016/06/29 14:17:58.046044 repair.go:278: repairing 1 chunks of untitled folder 3/.DS_Store
2016/06/29 14:17:58.046187 repair.go:278: repairing 1 chunks of untitled folder 3/1453756843707.jpg
2016/06/29 14:17:58.046279 repair.go:278: repairing 1 chunks of untitled folder 3/1453756944023.png
2016/06/29 14:18:03.047711 repair.go:278: repairing 1 chunks of 1453756944023.png
2016/06/29 14:27:00.767929 repair.go:301: failed to upload to host dataspace.ddns.net:9982: host is not accepting contracts
2016/06/29 14:28:04.404515 repair.go:301: failed to upload to host 108.61.96.209:9982: host is not accepting contracts
2016/06/29 14:28:07.273113 repair.go:301: failed to upload to host 97.107.52.227:9982: host is not accepting contracts
2016/06/29 14:28:09.368921 repair.go:301: failed to upload to host 198.211.103.40:9982: host is not accepting contracts
2016/06/29 14:28:10.044869 repair.go:301: failed to upload to host 216.84.97.66:9982: host is not accepting contracts
2016/06/29 14:28:10.044979 repair.go:301: failed to upload to host sia.tech.tedx.ee:9982: host is not accepting contracts
2016/06/29 14:28:10.045054 repair.go:301: failed to upload to host 82.220.99.82:9982: host is not accepting contracts
2016/06/29 14:28:10.045168 repair.go:301: failed to upload to host 52.26.159.234:9982: host did not accept revision: proposed file contract revision does not correctly deduct from the renter's valid proof output
2016/06/29 14:28:10.054188 repair.go:278: repairing 1 chunks of untitled folder 3/.DS_Store
2016/06/29 14:38:12.745538 repair.go:301: failed to upload to host 85.114.128.125:9982: contract has insufficient collateral to support upload
2016/06/29 14:38:12.753936 repair.go:301: failed to upload to host 138.201.13.159:9982: host did not accept revision: proposed file contract revision does not correctly add to the host's valid proof output
2016/06/29 14:38:12.754058 repair.go:301: failed to upload to host 146.90.15.59:9982: host did not accept revision: proposed file contract revision does not correctly add to the host's valid proof output
2016/06/29 14:38:12.754147 repair.go:301: failed to upload to host cmoser.duckdns.org:9982: host did not accept revision: proposed file contract revision does not correctly add to the host's valid proof output
2016/06/29 14:38:12.754231 repair.go:301: failed to upload to host alpha.sia.aspectron.com:9982: host did not accept revision: proposed file contract revision does not correctly add to the host's valid proof output
2016/06/29 14:38:12.754307 repair.go:301: failed to upload to host 112.74.97.188:9982: write tcp 192.168.1.4:54027->112.74.97.188:9982: write: broken pipe
2016/06/29 14:38:12.759319 repair.go:278: repairing 1 chunks of untitled folder 3/1453756843707.jpg
2016/06/29 14:45:01.255154 repair.go:278: repairing 1 chunks of untitled folder 3/1453756944023.png
2016/06/29 14:49:32.948063 repair.go:301: failed to upload to host 64.137.228.210:9982: host did not accept revision: proposed file contract revision does not correctly add to the host's valid proof output
2016/06/29 14:49:32.955792 repair.go:301: failed to upload to host 73.69.100.22:9982: host did not accept revision: proposed file contract revision does not correctly add to the host's valid proof output
2016/06/29 14:49:32.956028 repair.go:301: failed to upload to host 31.47.126.128:9982: host did not accept revision: proposed file contract revision does not correctly add to the host's valid proof output
2016/06/29 14:49:32.960272 repair.go:278: repairing 1 chunks of untitled folder 3/untitled folder/1443814339804.jpg
2016/06/29 14:50:23.557105 repair.go:301: failed to upload to host sia.internet-box.ch:9982: host did not accept revision: proposed file contract revision does not correctly add to the host's valid proof output
2016/06/29 14:50:23.567842 repair.go:278: repairing 1 chunks of untitled folder 3/untitled folder/1452160563683.jpg
2016/06/29 14:50:28.569621 repair.go:278: repairing 1 chunks of untitled folder 3/1453756944023.png
2016/06/29 15:00:55.822308 repair.go:301: failed to upload to host 85.114.128.125:9982: contract has insufficient collateral to support upload
2016/06/29 15:00:55.823803 repair.go:301: failed to upload to host 198.211.103.40:9982: host is not accepting contracts
2016/06/29 15:00:55.824147 repair.go:301: failed to upload to host dataspace.ddns.net:9982: host is not accepting contracts
2016/06/29 15:00:55.824247 repair.go:301: failed to upload to host 108.61.96.209:9982: host is not accepting contracts
2016/06/29 15:00:55.824319 repair.go:301: failed to upload to host 82.220.99.82:9982: host is not accepting contracts
2016/06/29 15:00:55.824403 repair.go:301: failed to upload to host sia.tech.tedx.ee:9982: host is not accepting contracts
2016/06/29 15:00:55.824493 repair.go:301: failed to upload to host 216.84.97.66:9982: host is not accepting contracts
2016/06/29 15:00:55.824554 repair.go:301: failed to upload to host 97.107.52.227:9982: host is not accepting contracts
2016/06/29 15:00:55.824687 repair.go:301: failed to upload to host 112.74.97.188:9982: write tcp 192.168.1.4:58631->112.74.97.188:9982: write: broken pipe
2016/06/29 15:00:55.824804 repair.go:301: failed to upload to host cmoser.duckdns.org:9982: write tcp 192.168.1.4:58999->66.41.100.180:9982: write: broken pipe
2016/06/29 15:00:55.824891 repair.go:301: failed to upload to host 31.47.126.128:9982: write tcp 192.168.1.4:59012->31.47.126.128:9982: write: broken pipe
2016/06/29 15:00:55.824976 repair.go:301: failed to upload to host 73.69.100.22:9982: write tcp 192.168.1.4:59023->73.69.100.22:9982: write: broken pipe
2016/06/29 15:00:55.825053 repair.go:301: failed to upload to host alpha.sia.aspectron.com:9982: write tcp 192.168.1.4:59037->24.37.11.107:9982: write: broken pipe
2016/06/29 15:00:55.825166 repair.go:301: failed to upload to host 52.26.159.234:9982: write tcp 192.168.1.4:59050->52.26.159.234:9982: write: broken pipe
2016/06/29 15:00:55.825257 repair.go:301: failed to upload to host 64.137.228.210:9982: write tcp 192.168.1.4:59067->64.137.228.210:9982: write: broken pipe
2016/06/29 15:00:55.828626 repair.go:278: repairing 1 chunks of untitled folder 3/untitled folder/1443814339804.jpg
2016/06/29 15:04:36.512388 repair.go:278: repairing 1 chunks of untitled folder 3/untitled folder/1452160563683.jpg
2016/06/29 15:08:12.419433 repair.go:301: failed to upload to host sia.internet-box.ch:9982: host did not accept revision: proposed file contract revision does not correctly add to the host's valid proof output
2016/06/29 15:08:12.427359 repair.go:301: failed to upload to host 138.201.13.159:9982: host did not accept revision: proposed file contract revision does not correctly add to the host's valid proof output
2016/06/29 15:08:12.434904 repair.go:278: repairing 1 chunks of 1453756944023.png
2016/06/29 15:08:12.435262 repair.go:278: repairing 1 chunks of untitled folder 3/.DS_Store
2016/06/29 15:08:16.981208 repair.go:301: failed to upload to host 146.90.15.59:9982: write tcp 192.168.1.4:59036->146.90.15.59:9982: write: broken pipe
2016/06/29 15:08:16.983794 repair.go:278: repairing 1 chunks of untitled folder 3/1453756843707.jpg
2016/06/29 15:08:21.984728 repair.go:278: repairing 1 chunks of untitled folder 3/.DS_Store
2016/06/29 15:18:29.406665 repair.go:301: failed to upload to host 85.114.128.125:9982: contract has insufficient collateral to support upload
2016/06/29 15:18:29.414974 repair.go:301: failed to upload to host sia.tech.tedx.ee:9982: host is not accepting contracts
2016/06/29 15:18:29.415136 repair.go:301: failed to upload to host 97.107.52.227:9982: host is not accepting contracts
2016/06/29 15:18:30.128570 repair.go:301: failed to upload to host 216.84.97.66:9982: host is not accepting contracts
2016/06/29 15:18:30.128750 repair.go:301: failed to upload to host 82.220.99.82:9982: host is not accepting contracts
2016/06/29 15:18:45.301939 repair.go:301: failed to upload to host 146.90.15.59:9982: host is not accepting contracts
2016/06/29 15:18:45.302130 repair.go:301: failed to upload to host 108.61.96.209:9982: host is not accepting contracts
2016/06/29 15:18:45.302286 repair.go:301: failed to upload to host 198.211.103.40:9982: host is not accepting contracts
2016/06/29 15:18:45.302443 repair.go:301: failed to upload to host dataspace.ddns.net:9982: host is not accepting contracts
2016/06/29 15:18:45.302599 repair.go:301: failed to upload to host 52.26.159.234:9982: host did not accept revision: proposed file contract revision does not correctly deduct from the renter's valid proof output
2016/06/29 15:18:45.302749 repair.go:301: failed to upload to host 112.74.97.188:9982: write tcp 192.168.1.4:58901->112.74.97.188:9982: write: broken pipe
2016/06/29 15:18:45.313530 repair.go:278: repairing 1 chunks of untitled folder 3/1453756843707.jpg
2016/06/29 15:20:51.987606 persist.go:409: STARTUP: Logging has started.
2016/06/29 15:20:57.011043 repair.go:278: repairing 1 chunks of untitled folder 3/untitled folder/1443814339804.jpg
2016/06/29 15:21:19.902653 persist.go:409: STARTUP: Logging has started.
2016/06/29 15:21:24.921403 repair.go:278: repairing 1 chunks of 1453756944023.png
2016/06/29 15:24:08.003893 repair.go:301: failed to upload to host 108.61.96.209:9982: host is not accepting contracts
2016/06/29 15:26:08.010707 repair.go:301: failed to upload to host 82.220.99.82:9982: host is not accepting contracts
2016/06/29 15:26:57.874425 persist.go:409: STARTUP: Logging has started.
2016/06/29 15:27:02.898624 repair.go:278: repairing 1 chunks of untitled folder 3/untitled folder/1443814339804.jpg
2016/06/29 15:37:28.917953 repair.go:301: failed to upload to host 85.114.128.125:9982: contract has insufficient collateral to support upload
2016/06/29 15:37:28.926348 repair.go:301: failed to upload to host dataspace.ddns.net:9982: host is not accepting contracts
2016/06/29 15:37:28.926496 repair.go:301: failed to upload to host 216.84.97.66:9982: host is not accepting contracts
2016/06/29 15:37:28.926574 repair.go:301: failed to upload to host 97.107.52.227:9982: host is not accepting contracts
2016/06/29 15:37:28.926640 repair.go:301: failed to upload to host sia.tech.tedx.ee:9982: host is not accepting contracts
2016/06/29 15:37:28.926701 repair.go:301: failed to upload to host 82.220.99.82:9982: host is not accepting contracts
2016/06/29 15:37:28.926769 repair.go:301: failed to upload to host 108.61.96.209:9982: host is not accepting contracts
2016/06/29 15:37:28.926845 repair.go:301: failed to upload to host 198.211.103.40:9982: host is not accepting contracts
2016/06/29 15:37:28.926916 repair.go:301: failed to upload to host 52.26.159.234:9982: write tcp 192.168.1.4:60316->52.26.159.234:9982: write: broken pipe
2016/06/29 15:37:28.926985 repair.go:301: failed to upload to host 112.74.97.188:9982: write tcp 192.168.1.4:60713->112.74.97.188:9982: write: broken pipe
2016/06/29 15:37:28.927044 repair.go:301: failed to upload to host alpha.sia.aspectron.com:9982: write tcp 192.168.1.4:60754->24.37.11.107:9982: write: broken pipe
2016/06/29 15:37:28.927131 repair.go:301: failed to upload to host 64.137.228.210:9982: write tcp 192.168.1.4:60801->64.137.228.210:9982: write: broken pipe
2016/06/29 15:37:28.927221 repair.go:301: failed to upload to host 31.47.126.128:9982: host did not accept revision: read tcp 192.168.1.4:60856->31.47.126.128:9982: read: connection reset by peer
2016/06/29 15:37:28.927264 repair.go:301: failed to upload to host 73.69.100.22:9982: write tcp 192.168.1.4:60891->73.69.100.22:9982: write: broken pipe
2016/06/29 15:37:28.935820 repair.go:278: repairing 1 chunks of untitled folder 3/untitled folder/1452160563683.jpg
2016/06/29 15:41:46.876775 repair.go:301: failed to upload to host cmoser.duckdns.org:9982: host did not accept revision: proposed file contract revision does not correctly add to the host's valid proof output
2016/06/29 15:41:46.881704 repair.go:278: repairing 1 chunks of 1453756944023.png
2016/06/29 15:41:46.881890 repair.go:278: repairing 1 chunks of untitled folder 3/.DS_Store
2016/06/29 15:45:57.853393 persist.go:409: STARTUP: Logging has started.
2016/06/29 15:46:02.874819 repair.go:278: repairing 1 chunks of untitled folder 3/1453756944023.png
2016/06/29 15:56:30.734397 repair.go:301: failed to upload to host 85.114.128.125:9982: contract has insufficient collateral to support upload
2016/06/29 15:56:30.744451 repair.go:301: failed to upload to host sia.tech.tedx.ee:9982: host is not accepting contracts
2016/06/29 15:56:30.744911 repair.go:301: failed to upload to host 108.61.96.209:9982: host is not accepting contracts
2016/06/29 15:56:30.745067 repair.go:301: failed to upload to host dataspace.ddns.net:9982: host is not accepting contracts
2016/06/29 15:56:30.745210 repair.go:301: failed to upload to host 216.84.97.66:9982: host is not accepting contracts
2016/06/29 15:56:30.745413 repair.go:301: failed to upload to host 97.107.52.227:9982: host is not accepting contracts
2016/06/29 15:56:30.745542 repair.go:301: failed to upload to host 82.220.99.82:9982: host is not accepting contracts
2016/06/29 15:56:30.745647 repair.go:301: failed to upload to host 198.211.103.40:9982: host is not accepting contracts
2016/06/29 15:56:30.745798 repair.go:301: failed to upload to host alpha.sia.aspectron.com:9982: write tcp 192.168.1.4:61648->24.37.11.107:9982: write: broken pipe
2016/06/29 15:56:30.745904 repair.go:301: failed to upload to host 112.74.97.188:9982: write tcp 192.168.1.4:61623->112.74.97.188:9982: write: broken pipe
2016/06/29 15:56:30.745992 repair.go:301: failed to upload to host 31.47.126.128:9982: write tcp 192.168.1.4:61681->31.47.126.128:9982: write: broken pipe
2016/06/29 15:56:30.746115 repair.go:301: failed to upload to host 64.137.228.210:9982: write tcp 192.168.1.4:61707->64.137.228.210:9982: write: broken pipe
2016/06/29 15:56:30.746202 repair.go:301: failed to upload to host 73.69.100.22:9982: write tcp 192.168.1.4:61834->73.69.100.22:9982: write: broken pipe
2016/06/29 15:56:30.746296 repair.go:301: failed to upload to host 52.26.159.234:9982: write tcp 192.168.1.4:61899->52.26.159.234:9982: write: broken pipe
2016/06/29 15:56:30.751745 repair.go:278: repairing 1 chunks of untitled folder 3/untitled folder/1443814339804.jpg
2016/06/29 15:56:30.751966 repair.go:278: repairing 1 chunks of untitled folder 3/untitled folder/1452160563683.jpg
2016/06/29 15:57:24.947363 persist.go:409: STARTUP: Logging has started.
2016/06/29 15:57:29.964162 repair.go:278: repairing 1 chunks of 1453756944023.png
2016/06/29 15:59:34.095385 repair.go:301: failed to upload to host 216.84.97.66:9982: host is not accepting contracts
2016/06/29 16:01:34.111273 repair.go:301: failed to upload to host dataspace.ddns.net:9982: host is not accepting contracts
2016/06/29 16:03:34.111730 repair.go:301: failed to upload to host sia.tech.tedx.ee:9982: host is not accepting contracts
2016/06/29 16:05:34.112470 repair.go:301: failed to upload to host 108.61.96.209:9982: host is not accepting contracts
2016/06/29 16:07:33.146213 repair.go:301: failed to upload to host 97.107.52.227:9982: host is not accepting contracts
2016/06/29 16:07:33.146365 repair.go:301: failed to upload to host 198.211.103.40:9982: host is not accepting contracts
2016/06/29 16:07:33.568469 repair.go:301: failed to upload to host 82.220.99.82:9982: host is not accepting contracts
2016/06/29 16:07:33.568612 repair.go:301: failed to upload to host 52.26.159.234:9982: host did not accept revision: proposed file contract revision does not correctly deduct from the renter's valid proof output
2016/06/29 16:07:33.582686 repair.go:278: repairing 1 chunks of untitled folder 3/.DS_Store
```

**Comment by hosiawak** [comment]
I have the exact same problem. I tried uploading a 25 MB, 650 MB and 40 KB files and even the small 40 KB file took hours to upload. What's the problem ?

    $ tail -n 100 -f renter.log
    2016/07/01 08:00:14.553904 repair.go:301: failed to upload to host chemsorly.selfhost.eu:9982: host did not accept transaction signature: host unable to add sector despite having the storage capacity to do so
    2016/07/01 08:00:14.553959 repair.go:301: failed to upload to host 178.202.162.162:9982: host did not accept transaction signature: host unable to add sector despite having the storage capacity to do so
    2016/07/01 08:00:14.554011 repair.go:301: failed to upload to host 213.73.138.240:9982: host did not accept revision: proposed file contract revision does not correctly add to the host's valid proof output
    2016/07/01 08:00:14.554061 repair.go:301: failed to upload to host 97.107.52.227:9982: host did not accept transaction signature: host unable to add sector despite having the storage capacity to do so
    2016/07/01 08:00:14.554108 repair.go:301: failed to upload to host 108.13.70.202:9982: host did not accept transaction signature: not enough storage remaining to accept sector
    2016/07/01 08:05:42.666913 repair.go:278: repairing 2 chunks of MOVIE.mp4
    2016/07/01 08:06:02.402045 repair.go:301: failed to upload to host 54.169.137.11:9982: contract has insufficient funds to support upload
    2016/07/01 08:06:02.402531 repair.go:301: failed to upload to host 213.73.138.240:9982: host did not accept revision: proposed file contract revision does not correctly add to the host's valid proof output
    2016/07/01 08:06:02.402624 repair.go:301: failed to upload to host 178.202.162.162:9982: host did not accept transaction signature: host unable to add sector despite having the storage capacity to do so
    2016/07/01 08:06:02.402686 repair.go:301: failed to upload to host chemsorly.selfhost.eu:9982: host did not accept transaction signature: host unable to add sector despite having the storage capacity to do so
    2016/07/01 08:06:02.402743 repair.go:301: failed to upload to host 97.107.52.227:9982: host did not accept transaction signature: host unable to add sector despite having the storage capacity to do so
    2016/07/01 08:06:02.402803 repair.go:301: failed to upload to host 80.234.37.94:9982: host did not accept revision: proposed file contract revision does not correctly add to the host's valid proof output
    2016/07/01 08:06:02.402848 repair.go:301: failed to upload to host 108.13.70.202:9982: host did not accept transaction signature: not enough storage remaining to accept sector

**Comment by Fornax96** [comment]
`2016/07/01 08:00:14.554011 repair.go:301: failed to upload to host 213.73.138.240:9982: host did not accept revision: proposed file contract revision does not correctly add to the host's valid proof output`

That's my host, is there anything I can do about it or is it just a renter issue?

**Comment by hosiawak** [comment]
New error I haven't seen above:

    2016/07/01 19:34:25.991032 repair.go:301: failed to upload to host 146.90.15.59:9982: couldn't read host's settings: could not decode type crypto.Signature: EOF

**Comment by DavidVorick** [comment]
@fornax96: is your host updated to the most recent version? When did you do the upgrade?

@hosiawak that error, as long as it's only happening once in a while, is likely just an unstable network connection. The connection should restart automatically from where it left off.

A lot of the hosts are disabling their contracts due to a bug that was recently merged into the master branch. After the next update, a lot of these issues should be resolved. We'll continue looking into the other error messages as well.

**Comment by DavidVorick** [comment]
fixed in v1.0.3
